pkgs/development/python-modules/python-djvulibre/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, djvulibre
+, ghostscript_headless
+, packaging
+, pkg-config
+, requests
+, setuptools
+, unittestCheckHook
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "python-djvulibre";
+  version = "0.9.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "FriedrichFroebel";
+    repo = "python-djvulibre";
+    rev = "${version}";
```

```suggestion
    rev = "${version}";
```
```suggestion
    rev = version;
```
> Unnecessary string conversions should be avoided.
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax